myhpi/tenca_django/migrations/0002_alter_hashentry_options_delete_legacyadminurl.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.0.7 on 2022-09-15 19:17
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("tenca_django", "0001_initial"),
```

Comment:
not sure if you actually want that in this PR

Reply:
It has to be in some PR, since it is created when running makemigrations

Reply:
you _could_ move it to a separate PR so that one is able to find it afterwards, as no one would assume that it is hidden in this PR. but I'm also fine with merging as-is

Reply:
I think that is not a problem since I made a specific commit for that change and don't squash